dev-python/requests-cache/requests-cache-0.5.2.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
```

Comment:
We don't need to bother with 3.6 on new ebuilds.

Reply:
Actually, please do not remove it where it is already tested. Some people still need that (particularly Sabayon).